pkgs/by-name/cm/cmatrix/package.nix

```
@@ -4,32 +4,38 @@
   fetchFromGitHub,
   autoreconfHook,
   ncurses,
+  versionCheckHook,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "cmatrix";
   version = "2.0";
 
   src = fetchFromGitHub {
     owner = "abishekvashok";
     repo = "cmatrix";
-    rev = "v${version}";
+    tag = "v${finalAttrs.version}";
     sha256 = "1h9jz4m4s5l8c3figaq46ja0km1gimrkfxm4dg7mf4s84icmasbm";
```

Sorry, I don't know how I missed this, but please change this to an sri hash


```suggestion
    hash = "sha256-...";
```